packages/expo-secure-store/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - [secure-store] Reset android isAuthenticating prompt flag when error is thrown ([#38132](https://github.com/expo/expo/pull/38132) by [@SYoder1](https://github.com/SYoder1))
 
+- [secure-store] use more specific error messages in authentication prompt errors ([#38340](https://github.com/expo/expo/pull/38340) by [@SYoder1](https://github.com/SYoder1))
```

Comment:
Actually one more thing here, please move the two entiries so that they are in the correct section: bug fix for the previous PR, feature for this one.

Thanks!